pkgs/tools/networking/whois/default.nix

```diff
@@ -18,8 +18,8 @@ stdenv.mkDerivation rec {
     for i in Makefile po/Makefile; do
       substituteInPlace $i --replace "prefix = /usr" "prefix = $out"
     done
-
-    substituteInPlace Makefile --replace "DEFS += HAVE_ICONV" "DEFS += HAVE_ICONV\nwhois_LDADD += -liconv"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
Can this be `(stdenv.isDarwin || stdenv.hostPlatform.isMusl)`? Thanks.

(related: #37012)

Reply:
👍